packages/mobile/src/notifications/SummaryNotification.tsx

```diff
@@ -1,69 +1,54 @@
-import SummaryNotification from '@celo/react-components/components/SummaryNotification'
-import fontStyles from '@celo/react-components/styles/fonts'
+import AggregatedRequestsMessagingCard from '@celo/react-components/components/AggregatedRequestsMessagingCard'
 import * as React from 'react'
-import i18n from 'src/i18n'
+import { Trans, useTranslation } from 'react-i18next'
+import { Namespaces } from 'src/i18n'
 
-import { StyleSheet, Text, View } from 'react-native'
-
-interface OwnProps<T> {
+interface Props<T> {
   title: string
-  icon: JSX.Element
+  detailsI18nKey: string
+  icon: React.ReactNode
   onReview: () => void
-  itemRenderer: (item: T, key: number) => JSX.Element
+  itemRenderer: (item: T, key: number) => React.ReactNode
   items: T[]
 }
 
-type Props<T> = OwnProps<T>
-
+// If changing this, you probably also need to update the translation files
+// which use it via detailsI18nKey
 const PREVIEW_SIZE = 2
 
-function getAdditionalItemsCount<T>(items: T[]) {
-  const total = items.length
-  if (total - PREVIEW_SIZE > 0) {
-    return (
-      <View>
-        <Text style={styles.moreWithCountText}>
-          {i18n.t('global:moreWithCount', { count: total - PREVIEW_SIZE + 1 })}
-        </Text>
-      </View>
-    )
+function getContext(count: number) {
+  if (count === 2) {
```

Comment:
do i care if this is a `switch` statement? no.
does this look exactly like a `switch` statement? yes.
is this comment pointless? yes.
😄

Reply:
Ahah, I remember starting with a `switch` statement, and then switched (no pun intended) to `if` thinking I'd make a comparison.